SwiftMessageBar/Message.swift

```diff
@@ -60,6 +62,13 @@ final class Message: UIView {
     let titleLabel = initTitle()
     let messageLabel = initMessage()
     
+    if languageDirection == .rightToLeft {
+      titleLabel.textAlignment = .right
```

Comment:
I was under the impression that textAlignemt `.natural` takes care of this? I'll test this later today.